frontend/src/scenes/plugins/optin/OptInPlugins.tsx

```diff
@@ -15,8 +15,8 @@ export function OptInPlugins(): JSX.Element {
     useEffect(() => {
         async function setStatus(): Promise<void> {
             try {
-                const response = await api.get('api/organizations/@current/plugins/status')
```

Comment:
Q: Context for this change?

Reply:
Deduplicating the API. Otherwise we had two endpoints doing the same thing slightly differently.